public/app/features/alerting/unified/rule-list/components/AlertRuleListItem.tsx

```diff
@@ -0,0 +1,257 @@
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { Alert, Icon, Stack, Text, TextLink, useStyles2 } from '@grafana/ui';
+import { Trans } from 'app/core/internationalization';
+import { CombinedRule, CombinedRuleNamespace, RuleHealth } from 'app/types/unified-alerting';
+import { Labels, PromAlertingRuleState } from 'app/types/unified-alerting-dto';
+
+import { logError } from '../../Analytics';
+import { MetaText } from '../../components/MetaText';
+import { ProvisioningBadge } from '../../components/Provisioning';
+import { PluginOriginBadge } from '../../plugins/PluginOriginBadge';
+import { GRAFANA_RULES_SOURCE_NAME } from '../../utils/datasource';
+import { labelsSize } from '../../utils/labels';
+import { createContactPointLink } from '../../utils/misc';
+import { RulePluginOrigin } from '../../utils/rules';
+
+import { ListItem } from './ListItem';
+import { RuleListIcon } from './RuleListIcon';
+import { calculateNextEvaluationEstimate } from './util';
+
+interface AlertRuleListItemProps {
+  name: string;
+  href: string;
+  summary?: string;
+  error?: string;
+  state?: PromAlertingRuleState;
+  isPaused?: boolean;
+  health?: RuleHealth;
+  isProvisioned?: boolean;
+  lastEvaluation?: string;
+  evaluationInterval?: string;
+  labels?: Labels;
+  instancesCount?: number;
+  namespace?: CombinedRuleNamespace;
+  group?: string;
+  // used for alert rules that use simplified routing
+  contactPoint?: string;
+  actions?: ReactNode;
+  origin?: RulePluginOrigin;
+}
+
+export const AlertRuleListItem = (props: AlertRuleListItemProps) => {
+  const {
+    name,
+    summary,
+    state,
+    health,
+    error,
+    href,
+    isProvisioned,
+    lastEvaluation,
+    evaluationInterval,
+    isPaused = false,
+    instancesCount = 0,
+    namespace,
+    group,
+    contactPoint,
+    labels,
+    origin,
+    actions = null,
+  } = props;
+
+  const metadata: ReactNode[] = [];
+  if (namespace && group) {
+    metadata.push(
+      <Text color="secondary" variant="bodySmall">
+        <RuleLocation namespace={namespace} group={group} />
+      </Text>
+    );
+  }
+
+  if (!isPaused) {
+    if (lastEvaluation && evaluationInterval) {
+      metadata.push(
+        <EvaluationMetadata lastEvaluation={lastEvaluation} evaluationInterval={evaluationInterval} state={state} />
+      );
+    }
+
+    if (instancesCount) {
+      metadata.push(
+        <MetaText icon="layers-alt">
+          <TextLink href={href + '?tab=instances'} variant="bodySmall" color="primary" inline={false}>
+            {pluralize('instance', instancesCount, true)}
+          </TextLink>
+        </MetaText>
+      );
+    }
+  }
+
+  if (!isEmpty(labels)) {
+    metadata.push(
+      <MetaText icon="tag-alt">
+        <TextLink href={href} variant="bodySmall" color="primary" inline={false}>
+          {pluralize('label', labelsSize(labels), true)}
+        </TextLink>
+      </MetaText>
+    );
+  }
+
+  if (!isPaused && contactPoint) {
```

Comment:
maybe we could also render the contact point even if the alert rule is paused? But then we could use a different color or something to make clear that is paused and its not going to receive anything?